src/renderer/components/data-settings/data-settings.js

```diff
@@ -1008,6 +922,24 @@ export default defineComponent({
       return convertedData
     },
 
+    promptAndWriteToFile: async function (saveOptions, content, successMessageKey) {
```

Comment:
Or partial key, point is ensure other readers know the variable is not a **full** key 
```suggestion
    promptAndWriteToFile: async function (saveOptions, content, successMessageKeySuffix) {
```